Sleep Tracker/Sleep Tracker/View/SleepSetupView/Subviews/RoundedAlarmView.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+/// A rectangular rounded view with an alarm clock image in the center.
+/// When clicked, change its opacity property with animation.
+struct RoundedAlarmView: View {
+    /// A variable boolean value that visually changes the rectangular rounded view.
+    /// When takes the value `true` rectangular rounded view has no transparency
+    /// And when takes the value `false` rectangular rounded view has transparency.
```

Comment:
Something like that:
```suggestion
    /// A boolean value that indicates whether the alarm is animated.
    /// When `true` alarm is animated and when `false` alarm is not animated.

Reply:
fixed